web_app_drawer/views/web.xml

```diff
@@ -0,0 +1,238 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016 LasLabs Inc.
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<odoo>
+
+	<template id="web.webclient_bootstrap" name="Webclient Mobile">
+		<t t-call="web.layout">
+			
+			<t t-set="head">
+				
+				<meta charset="utf-8" />
+				<meta http-equiv="cleartype" content="on" />
+				<meta name="MobileOptimized" content="320" />
+				<meta name="HandheldFriendly" content="True" />
+				<meta name="apple-mobile-web-app-capable" content="yes" />
+				<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
+					
+				<t t-call-assets="web.assets_common"/>
+				
+				<script type="application/javascript">
+					odoo.define('web.csrf', function (require) {
+						var token = "<t t-esc="request.csrf_token(None)"/>";
+						require('web.core').csrf_token = token;
+						require('qweb').default_dict.csrf_token = token;
+					});
+					odoo.db_info = <t t-raw="db_info"/>;
+				</script>
+				
+				<t t-call-assets="web.assets_backend"/>
+				&lt;!--[if lte IE 9]&gt; <link rel="stylesheet" href="/web/static/src/css/ie.css"/> &lt;![endif]--&gt;
+				
+				<script type="application/javascript">
+					$(function() {
+						<t t-if="init">
+							<t t-raw="init"/>
+						</t>
+						<t t-if="not init">
+							odoo.define('web.web_client', function (require) {
+								var WebClient = require('web.WebClient');
+								var web_client = new WebClient();
+								web_client.setElement($(document.body));
+								web_client.start();
+								return web_client;
+							});
+						</t>
+					});
+				</script>
+
+			</t>
+			
+			<t t-set="body_classname" t-value="'drawer drawer--left'" />
+				
+			<header role="banner">
+				<nav id="odooAppDrawer" class="css-app-drawer-nav drawer-nav" role="navigation">
+					<t t-call="web.menu" />
+				</nav>
+			</header>
+				
+			<main role="main">
+				
+				<nav class="navbar navbar-default css-main-nav"
+					 role="navigation"
+					 groups="base.group_user,base.group_portal"
+					 >
+					<div class="container-fluid">
+						
+						<div class="navbar-header">
+							
+							<a href="#"
+							   class="drawer-toggle pull-left navbar-collapse collapse btn btn-default css-app-drawer-toggle"
+							   accesskey="A"
```

Comment:
I'm installing a Linux VM now; you're on GNOME right @Yajo ? And you're Windows @pedrobaeza ?